doc/userguide/src/CreatingTestData/AdvancedFeatures.rst

```diff
@@ -144,44 +144,40 @@ occurs, because they are normally engaged in important clean-up
 activities. If necessary, it is possible to interrupt also these
 keywords with `user keyword timeouts`_.
 
-.. table:: Test timeout examples
-   :class: example
-
-   ============  =========  =======  =======
-     Setting       Value     Value    Value
-   ============  =========  =======  =======
-   Test Timeout  2 minutes
-   ============  =========  =======  =======
-
-.. table::
-   :class: example
-
-   ===============  ===============  ========================================  ==========================  ==================
-      Test Case         Action                      Argument                           Argument                 Argument
-   ===============  ===============  ========================================  ==========================  ==================
-   Default Timeout  [Documentation]  Timeout from the Setting table is used
-   \                Some Keyword     argument
-   \
-   Override         [Documentation]  Override default, use 10 seconds timeout
-   \                [Timeout]        10
-   \                Some Keyword     argument
-   \
-   Custom Message   [Documentation]  Override default and use custom message
-   \                [Timeout]        1min 10s                                  This is my custom error.    It continues here.
-   \                Some Keyword     argument
-   \
-   Variables        [Documentation]  It is possible to use variables too
-   \                [Timeout]        ${TIMEOUT}
-   \                Some Keyword     argument
-   \
-   No Timeout       [Documentation]  Empty timeout means no timeout even when  Test Timeout has been used
-   \                [Timeout]
-   \                Some Keyword     argument
-   \
-   No Timeout 2     [Documentation]  Empty timeout using NONE, works           as well
-   \                [Timeout]        NONE
-   \                Some Keyword     argument
-   ===============  ===============  ========================================  ==========================  ==================
+.. sourcecode:: robotframework
+
+   *** Settings ***
+   Test Timeout    2 minutes
+
+   *** Test Cases ***
+   Default Timeout
+       [Documentation]    Timeout from the Setting table is used
+       Some Keyword    argument
+   
+   Override
+       [Documentation]    Override default, use 10 seconds timeout
+       [Timeout]    10
+       Some Keyword    argument
+   
+   Custom Message
+       [Documentation]    Override default and use custom message
+       [Timeout]    1min 10s    This is my custom error
+       Some Keyword    argument
+   
+   Variables
+       [Documentation]    It is possible to use variables too
+       [Timeout]    ${TIMEOUT}
+       Some Keyword    argument
+   
+   No Timeout
+       [Documentation]    Empty timeout means no timeout even when Test Timeout has been used
+       [Timeout]
+       Some Keyword    argument
+   
+   No Timeout 2
+       [Documentation]    Empty timeout using NONE, works as well
```

Comment:
Something like "Disable timeout with NONE works too and is more explicit that using empty timeout." would be a better documentation. The problem is obviously already in the original.